zmq-sys/build.rs

```diff
@@ -3,6 +3,27 @@ extern crate metadeps;
 use std::env;
 use std::path::Path;
 
+fn verify_windows_build_environment() {
+    let target = env::var("TARGET").ok().unwrap();
+    println!("cargo:target={:?}", target);
```

Comment:
I cannot answer you really, just wanted to determine if the target is Windows, and this got the job done.
I just used the reference to `cargo:target` to print some meaningful information for the user in the event that the code panics.
Will remove `fn verify_windows_build_environment()` altogether as suggested to default back to the original, so this will be resolved.